devices/IWearRemapper/src/IWearRemapper.cpp

```diff
@@ -135,15 +136,38 @@ bool IWearRemapper::open(yarp::os::Searchable& config)
     }
 
     // RPC ports
-    if (!(config.check("wearableRPCPorts") && config.find("wearableRPCPorts").isList())) {
-        yError() << logPrefix << "wearableRPCPorts option does not exist or it is not a list";
-        return false;
+    if (!(config.check("useRPC"))) {
+        yWarning() << logPrefix << "OPTIONAL parameter useRPC NOT found";
     }
-    yarp::os::Bottle* inputRPCPortsNamesList = config.find("wearableRPCPorts").asList();
-    for (unsigned i = 0; i < inputRPCPortsNamesList->size(); ++i) {
-        if (!inputRPCPortsNamesList->get(i).isString()) {
-            yError() << logPrefix << "ith entry of wearableRPCPorts list is not a string";
-            return false;
+    else {
+        if (!config.find("useRPC").isBool()) {
+            yError() << logPrefix << "useRPC option is not a bool";
+        }
+        pImpl->useRPC = config.find("useRPC").asBool();
+        yWarning() << "useRPC set to" << pImpl->useRPC;
```

Comment:
That was to use the same type of messages used for [`XsensSuit`](https://github.com/robotology-playground/wearables/blob/master/devices/XsensSuit/src/XsensSuit.cpp#L667).

However, I agree on removing the messages since I think not using `useRPC` parameter could be standard usage.

Reply:
As I told to Yeshi, do not consider those devices as the ground truth, they didn't pass through a detailed review process and minor things like this one have not been taken into account.

Reply:
fixed https://github.com/robotology-playground/wearables/pull/24/commits/ef5c8460c53bfb452d36ac4ac9f38030cff6721c